.github/workflows/cmake.yml

```diff
@@ -36,8 +36,7 @@ jobs:
           - os: 'ubuntu-20.04'
             build_type: Release
             compiler: {c: gcc-7, cxx: g++-7}
-
-    runs-on: ${{matrix.os}}
+    runs-on: ${{ (matrix.os == 'ubuntu-22.04' && github.repository_owner == 'oneapi-src') && 'intel-ubuntu-22.04' || matrix.os }}
 
```

Comment:
I think this build is failing now because of
```
WARNING: The scripts sphinx-apidoc, sphinx-autogen, sphinx-build and sphinx-quickstart are installed in '/home/ubuntu/.local/bin' which is not on PATH.
```

the fix should be simple: `echo "$HOME/.local/bin" >> $GITHUB_PATH`